scripts/docs/generate_asset_documentation.py

```diff
@@ -72,23 +158,31 @@ def add_intro(doc, asset):
         if "languages" in asset["properties"]:
             doc.add_paragraph("**Languages**: " + asset["properties"]["languages"])
 
+    if "tags" in asset:
+        doc.add_heading("Tags ", level=3)
+        tags = []
+        for tag, value in asset["tags"].items():
+            tags.append("`{}` ".format(tag if len(str(value)) == 0 else (tag + " : " + str(value))))
+        tags.sort()
```

Comment:
`tags.sort(key=lambda x: x[0].lower())` (to avoid case-insensitivty although I'm not sure if tags can be lower/upper-case, this will also need to be done for references later too

Reply:
it didn't work, good point, I'll follow up on that